src/librustc_apfloat/ieee.rs

```diff
@@ -1199,8 +1201,8 @@ impl<S: Semantics> Float for IeeeFloat<S> {
         }
 
         // Handle a leading minus sign.
-        let minus = s.starts_with("-");
-        if minus || s.starts_with("+") {
+        let minus = s.starts_with('-');
+        if minus || s.starts_with('+') {
```

Comment:
This is the only change in this file that we should keep IMO (I guess I wasn't aware `starts_with` could take a `char`, when I did the port).